### DISCUSSION TITLE
Working with custom tags (non-boolean)

### DISCUSSION THREAD

**Discussion by JGRSilva** [webdriverio/webdriverio#12924] [Q&A]
In the project I'm working on, we would like to create custom tags with values, for example: platformVersion11, platformVersion12. We want the capabilities in BrowserStack to be correctly configured according to the number following the word 'platformVersion'. 
This configuration of capabilities is done in the beforeSession function of wdio.conf.ts. Is it possible to read this tag in wdio.conf.ts to extract its name?
I know cucumberOpts.tags has a similar behavior, but it only adds the tags on a execution list, in my case I want to manipulate the name of these tags.

**Comment by christian-bromann** [comment]
@JGRSilva you can assign custom tags in the capabilities, e.g. `"my:tag": "foobar"` and read it in the test via: `browser.capabilities['my:tag']`. Note, it is important to define custom capabilities with a `:` in the name.
